sklearn/model_selection/_search.py

```diff
@@ -999,3 +1004,356 @@ def fit(self, X, y=None, labels=None):
                                           self.n_iter,
                                           random_state=self.random_state)
         return self._fit(X, y, labels, sampled_params)
+
+
+class SequentialSearchCV(BaseSearchCV):
+    """Search over hyperparameters using Gaussian process.
```

Comment:
"Search over hyperparameters based on a surrogate model" (would be more generic if we plan to provide alternatives to GPs in the future)